interop/src/lib.rs

```diff
@@ -9,6 +10,10 @@ pub mod pb {
     include!(concat!(env!("OUT_DIR"), "/grpc.testing.rs"));
 }
 
+pub mod grpc_pb {
+    grpc::include_proto!("", "test");
```

Comment:
This `""` feels slightly awkward as well

Reply:
I think you can have macros take different amounts of arguments so you can have a macro that is `include_proto!("test")` and then if you do `include_proto!("dir", "test")` it will hit a different branch in the macro. I think that is enough to solve this problem?